hardware/raspberrypi/bcm2711_bootloader_config.md

```diff
@@ -119,14 +119,59 @@ On earlier models the serial number is used as the prefix, however, on Pi4 the M
 * 0 - Use the serial number e.g. "9ffefdef/"
 * 1 - Use the string specified by TFTP_PREFIX_STR
 * 2 - Use the MAC address e.g. "DC-A6-32-01-36-C2/"
-Default: 0
+Default: 0  
 Version: stable/pieeprom-2020-01-17.bin  
 
 ### TFTP_PREFIX_STR
 Specify the custom directory prefix string used when TFTP_PREFIX is set to 1. For example:- TFTP_PREFIX_STR=tftp_test/
-Default: ""
+Default: ""  
 Version: stable/pieeprom-2020-01-17.bin  
 
+### PXE_OPTION43
+Overrides the PXE Option43 match string with a different string. It's normally better to apply customizations to the DHCP server than change the client behaviour but this option is provided in case that's not possible.
+Default: "Raspberry Pi Boot"  
+Version: stable/pieeprom-2020-03-19.bin  
+
+### DHCP_OPTION97
+In earlier releases the client GUID (Option97) was just the serial number repeated 4 times. By default, the new GUID format is
+the concatenation of the fourcc for RPi4 (0x80017590), the board revision (e.g. 0x00c03111) (4-bytes), the least significant 4 bytes of the mac address and the 4-byte serial number.
```

Comment:
Is this definitely a fourcc? https://codebeautify.org/hex-string-converter suggests that it doesn't decode to an ASCII value? Is it just a "unique number"?

Reply:
done - copied the pointer value by accident